routes/articleRoutes.js

```diff
@@ -34,14 +41,16 @@ router.route('/')
     createArticle
   );
 
-
 router.route('/tag')
   .patch(
     verifyToken,
     checkIfTagIsString,
     tagArticle
   );
 
+router.route('/categories')
```

Comment:
This appears twice due to the rebase. Please remove one of them.

Reply:
ok. Removing it right away